contracts/REXUniswapV3Market.sol

```diff
@@ -451,6 +686,15 @@ contract REXUniswapV3Market is Ownable, SuperAppBase, Initializable{
         );
     }
 
+    /**
+     * @notice Called by the Superfluid framework after a new flow agreement has been created
```

Comment:
Nice way to say it 👍 Include this phrase `Called by the Superfluid framework` consistently across all the `before/afterAgreementXxxxx` methods, the `@notice` on those methods starts off inconsistently, this is a good way to start the notice.